cygnus-common/src/main/java/com/telefonica/iot/cygnus/utils/DataUtils.java

```diff
@@ -0,0 +1,89 @@
+/**
+ * 
```

Comment:
License header missing (please check in all new files in this PR)

Reply:
Done